### DISCUSSION TITLE
`inotify` integration

### DISCUSSION THREAD

**Discussion by TheChymera** [GothenburgBitFactory/taskwarrior#3148] [Q&A]
Is there any (automatic) way to integrate taskwarrior with notification demons — in order to get notifications on screen for urgent tasks?
As it stands it's a great app, but I have to always check it in a new window...

I'm thinking of setting up a cron job which checks taskwarrior outputs and sends notifications via `notify-send` depending on urgency, but I'm wondering if there's perhaps a better way to do this.

**Comment by djmitche** [comment]
I have some (possibly outdated) notes [here](https://gist.github.com/djmitche/dd7c9f257306e6b8957759c4d5265cc9) that might help.

**Comment by bergercookie** [comment]
@TheChymera one alternative is to use a synchronization service supported by [syncall](https://github.com/bergercookie/syncall) so that your tasks end up in e.g., your google calendar or in your caldav server. This way you'll get notifications from that service and you'll also be able to manage your tasks from taskwarrior.
